.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  release:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        scala_version: &scala_versions
+          - 2.12.12
+          - 2.13.4
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with: &cache_options
```

Comment:
I have not seen this syntax before; it seems to let you re-use options? I can't find documentation on these `&` and `*` symbols for GHA; do you have any links?

Reply:
Ah.. it seems these aren't currently supported yet. It's fialing to start the CI: https://github.com/lucidsoftware/xtract/actions/runs/435208437

`The workflow is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'scala_versions'`

Reply:
Ah.. it seems these aren't currently supported yet. It's failing to start the CI: https://github.com/lucidsoftware/xtract/actions/runs/435208437

`The workflow is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'scala_versions'`

Reply:
:( that's unfortunate. I was trying to avoid having to repeat the same config in multiple places. My biggest gripe with GHA is it can sometimes be very verbose, and there isn't a good way to avoid duplication.